R/data.R

```diff
@@ -78,7 +78,10 @@ spim_data_single <- function(date, region, model_type, rtm, serology,
   ## are too likely to be back-filled to be reliable
   i <- seq(to = nrow(data), length.out = trim_deaths)
   data[i, c("deaths", "deaths_hosp", "deaths_comm",
-            "deaths_carehomes",  "deaths_non_hosp")] <- NA
+            "deaths_carehomes",  "deaths_non_hosp",
```

Comment:
might just be easier to use a grep here?

Reply:
Not a huge fan of grep for these things, tbh

Reply:
I can see the benefit of both approaches. How about this? Makes it easier to modify, should we decide to change the age bands, but still very explicit as to what we are NAing

`deaths_hosp_age <- paste0("deaths_hosp_", c(0, seq(50, 80, 5)),
                            "_", c(seq(49, 79, 5), 120))
  deaths_hosp_age <- gsub("120", "plus", deaths_hosp_age)
  i <- seq(to = nrow(data), length.out = trim_deaths)
  data[i, c("deaths", "deaths_hosp", "deaths_comm",
            "deaths_carehomes",  "deaths_non_hosp",
            deaths_hosp_age)] <- NA`

Reply:
Happy with doing it that way instead!